lib/cadet/course/group.ex

```diff
@@ -20,5 +20,7 @@ defmodule Cadet.Course.Group do
   def changeset(group, attrs \\ %{}) do
     group
     |> cast(attrs, @required_fields ++ @optional_fields)
+    |> add_belongs_to_id_from_model(:leader, attrs)
```

Comment:
Why not combine these two into `|> add_belongs_to_id_from_model([:leader, :mentor], attrs)`? Read the documentation please, `add_belongs_to_id_from_model/3` has been painstakingly documented by @tuesmiddt, don't let it go to waste.